packages/plugins/graphql/server/src/services/builders/resolvers/query.ts

```diff
@@ -16,6 +16,9 @@ export default ({ strapi }: Context) => ({
           auth: ctx?.state?.auth,
         });
 
+        // Store the root query args in context so they can be accessed deeply down the tree (dz, components, etc..)
+        ctx.rootQueryArgs = args;
```

Comment:
ℹ️ I'm not sure this is the most elegant solution but I was unable to find another way, lmk what you think or if you have other suggestions

Reply:
Nothing comes to me; but maybe add a comment that this is for internal use only to dissuade anyone from using it so we can change it later if necessary.